plugins/hypervisors/kvm/pom.xml

```diff
@@ -45,7 +38,7 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.libvirt</groupId>
+      <groupId>com.github.K0zka</groupId>
```

Comment:
@kishankavala yes that is possible. I have to see how that could work together with the apache infrastructure, but I will be working on it. I will close this PR for now.